tests/ParserTest.elm

```diff
@@ -439,408 +440,347 @@ expr =
             [ ( "lambda"
               , [ ( "works with single argument"
                   , "\\x -> x + 1"
-                  , Just
-                        (AST.Frontend.lambda
-                            [ VarName "x" ]
-                            (Plus
-                                (Argument (VarName "x"))
-                                (Literal (Int 1))
-                            )
-                        )
+                  , Just (located { end = { col = 12, row = 1 }, start = { col = 1, row = 1 } } (Lambda { arguments = [ VarName "x" ], body = located { end = { col = 12, row = 1 }, start = { col = 7, row = 1 } } (Plus (located { end = { col = 8, row = 1 }, start = { col = 7, row = 1 } } (Argument (VarName "x"))) (located { end = { col = 12, row = 1 }, start = { col = 11, row = 1 } } (Literal (Int 1)))) }))
```

Comment:
I don't see a nice way to get rid of the position info here. :thinking: The closest that comes to mind is having the Frontend.Expr type take a parameter of what its children are ... and then mapping from `Frontend.Expr Frontend.LocatedExpr` into `Frontend.Expr Frontend.Expr` (which btw won't typecheck without using something like recursion schemes - https://blog.sumtypeofway.com/an-introduction-to-recursion-schemes/) ... I can try that in my own later. Let's sleep on it :man_shrugging: